packages/components/Checkbox/src/Checkbox.settings.ts

```diff
@@ -46,9 +45,10 @@ export const settings: IComposeSettings<ICheckboxType> = [
     content: {
       variant: 'bodyStandard',
       style: {
-        marginTop: 1,
         borderStyle: 'dotted',
-        borderWidth: 1
+        borderWidth: 1,
+        alignSelf: 'baseline',
+        marginTop: 3
```

Comment:
>3 [](start = 19, length = 1)

Just curious why is this needed?

Reply:
+1, does `alignSelf: 'baseline'` have any effect? I tried testing it out but I don't see a difference between auto (default) and baseline.

Reply:
You may be right, in my initial testing I thought it had an effect, but testing it again now, it doesn't look like it. I will remove it